### DISCUSSION TITLE
Disable selection checkbox in some rows in table component

### DISCUSSION THREAD

**Discussion by manoj000** [IBM/carbon-components-angular#1727] [Q&A] [answered]
How to disable a particular check box of a row in the table component, is that possible in the current implementation?

I am having the following scenario, where the user can select the checkbox and perform operations and they are not allowed to do so in some. How to do Implement this, an example would be of great help.

**Comment by manoj000** [comment]
@cal-smith

**Comment by cal-smith** [comment, marked as answer]
This isn't supported in the model at the moment. You could try using a custom template for the first item, or using the body and row directives directly and building the table you need from that. You can check the [story here](https://angular.carbondesignsystem.com/?path=/story/components-table--from-components) for an example of how the base table components can be used.
